### PR TITLE
Allow accessories to be retrieved from the FindMyiPhone service

### PR DESCRIPTION
**Pull request by koon64**
## Proposed change

This will allow users to access their accessories (ex. Airpods) in the FindMyiPhone service.

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New service (thank you!)
- [x] New feature (which adds functionality to an existing service)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests
- [ ] Documentation or code sample

## Additional information

Functionality is the same but there are more devices listed.

## Checklist

- [x] The code change is tested and works locally.
- [x] Local tests pass.
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works. 
*same functionality as before*

If user exposed functionality or configuration variables are added/changed:

None

- [ ] Documentation added/updated to README

### CODE REVIEW

File: pyicloud/services/findmyiphone.py [hunk 1]
```diff
@@ -43,6 +43,7 @@ def refresh_client(self):
                         "fmly": self.with_family,
                         "shouldLocate": True,
                         "selectedDevice": "all",
+                        "deviceListVersion": 1
```
Comment (Quentame) [inline pyicloud/services/findmyiphone.py:46]: add a coma at the en of line to fix `black` formatting

Comment (koon64) [reply pyicloud/services/findmyiphone.py]: Done

**Comment by Quentame** [comment]
May you give me your devices `deviceClass` ?

I need it to update this : 

https://github.com/home-assistant/core/blob/f8f05c4d0d491e8d4c9b17a885fd71f1498017dd/homeassistant/components/icloud/device_tracker.py#L140-L148

**Comment by koon64** [comment]
The `deviceClass` is `Accessory`

**Comment by Quentame** [comment]
> The `deviceClass` is `Accessory`

For everything ? Airpods, Watch, HomePod ...

**Comment by koon64** [comment]
No just for the Airpods. The Apple watch is `Watch` but that was already included in the previous device list. 
As for the HomePod, I just checked with someone that has a HomePod and he said it didn't show up in his devices on the iCloud Find My iPhone website. However I found this which means it showed up for someone else on Reddit. 

[https://www.reddit.com/r/HomePod/comments/azhr84/homepod_custom_name_not_showing_in_icloud/](https://www.reddit.com/r/HomePod/comments/azhr84/homepod_custom_name_not_showing_in_icloud/)

**Comment by Quentame** [comment]
Thanks

**Comment by koon64** [comment]
I kept looking and found this which you already commented on so prob already know this but it says the HomePod class is `HomePod`

[https://gist.github.com/insidegui/a18124c0c573a4eb656f5c485ea7dae4](https://gist.github.com/insidegui/a18124c0c573a4eb656f5c485ea7dae4)

**Pull request merged**